src/core.js

```diff
@@ -154,8 +154,8 @@ define([
                     return; // abort
                 }
                 _this.ensureCurrentMugIsSaved(function () {
-                    if (!_.isUndefined(window.analytics)) {
-                        window.analytics.track("Clicked Save in Formbuilder");
+                    if (!_.isUndefined(window._kmq)) {
+                        window._kmq.push(["record", "Clicked Save in Formbuilder"]);
```

Comment:
I agree a generic API with the same interface makes sense, I'm going to work on that. Thanks for the heads up about copy-paste-dev. I'll update this PR or close it when needed.